README.md

```diff
@@ -927,7 +927,7 @@ Other Style Guides
     });
 
     // good
-    [1, 2, 3].map(number => `A string containing the ${number}.`);
+    [1, 2, 3].map(number => `A string containing the ${number + 1}.`);
```

Comment:
I'm not sure this is an error; the examples don't all have to do identical things.